### PR TITLE
fix(@schematics/angular): add missing icons in ng-new template

### LINKED ISSUE #26257: @schematics/angular: `ng new` generated app has no icons
### Command

new

### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

The new buttons are intended to show an icon, but they just show the icon text.

Screenshot 2023-11-07 at 10 49 06

### Minimal Reproduction

```
npx -p @angular/cli@next ng new book-rating --standalone --routing --style=scss --no-ssr
```

### Your Environment

```text
Angular 17.0.0-rc.4
```

### PR DESCRIPTION
**Pull request by alan-agius4**
This commit adds a number of missing icons.

Closes #26257

Screenshot 2023-11-07 at 11 36 20

### CODE REVIEW

**Comment by nartc** [comment]
Should we also update the favicon to be the new logo?

File: packages/schematics/angular/application/files/common-files/src/app/app.component.html.template [hunk 1]
```diff
@@ -142,13 +142,13 @@
...
+  .pill-group .pill:nth-child(6n + 3),
   .pill-group .pill:nth-child(6n + 4),
   .pill-group .pill:nth-child(6n + 5) {
     --pill-accent: var(--hot-red);
   }
 
-  .open-in-new-icon {
+  .pill-group svg {
     margin-inline-start: 0.25rem;
     font-size: 1rem;
```
Comment (twerske) [inline packages/schematics/angular/application/files/common-files/src/app/app.component.html.template:153]: I think this now needs to be smaller, the icons are too large.

**Review by twerske — APPROVED** [review]
Could we use @for() to make the pill list shorter? Maybe that's for v18 :)

File: packages/schematics/angular/application/files/common-files/src/app/app.component.html.template (hunk 1 above)
Comment (alan-agius4) [reply packages/schematics/angular/application/files/common-files/src/app/app.component.html.template:153]: Screenshot 2023-11-07 at 16 31 30

File: packages/schematics/angular/application/files/common-files/src/app/app.component.html.template [hunk 2]
```diff
@@ -234,65 +225,37 @@
...
-        >
-          <span> Explore the Docs </span>
-          <i class="material-icons-outlined open-in-new-icon" aria-hidden="true"
-            >open_in_new</i
+        @for (item of [
+          { title: 'Explore the Docs', link: 'https://angular.dev' },
+          { title: 'Learn with Tutorials', link: 'https://angular.dev/tutorials' },
+          { title: 'CLI Docs', link: 'https://angular.dev/cli' },
+          { title: 'Angular Language Service', link: 'https://angular.dev/tools/language-service' },
+          { title: 'Angular DevTools', link: 'https://angular.dev/tools/dev-tools' },
```
Comment (dgp1130) [inline packages/schematics/angular/application/files/common-files/src/app/app.component.html.template:237]: Should be `/tools/devtools`

File: packages/schematics/angular/application/files/common-files/src/app/app.component.html.template [hunk 3]
```diff
@@ -234,65 +225,37 @@
...
-          target="_blank"
-          rel="noopener"
-        >
-          <span> Explore the Docs </span>
-          <i class="material-icons-outlined open-in-new-icon" aria-hidden="true"
-            >open_in_new</i
+        @for (item of [
+          { title: 'Explore the Docs', link: 'https://angular.dev' },
+          { title: 'Learn with Tutorials', link: 'https://angular.dev/tutorials' },
+          { title: 'CLI Docs', link: 'https://angular.dev/cli' },
```
Comment (dgp1130) [inline packages/schematics/angular/application/files/common-files/src/app/app.component.html.template:235]: Should be `/tools/cli` for the guide (`/cli` is reference docs).

**Review by dgp1130 — APPROVED** [review]

**Pull request merged**